### PR TITLE
docs: replace/canset cardinality documentation (DEV-1564 & DEV-1563)

### PR DESCRIPTION
**Pull request by seakayone**
### Task Description/Number

closes: DEV-1564 & DEV-1563

* Adds documentation for added and updated endpoints.
* Renames `ReplaceClassCardinalitiesRequestV2`
* Adds missing copyright headers

### Basic Requirements

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

### PR Type

What kind of change does this PR introduce?

- [ ] Bugfix: represents bug fixes
- [x] Refactor: represents production code refactoring
- [ ] Feature: represents a new feature
- [x] Documentation: documentation changes (no production code change)
- [ ] Chore: maintenance tasks (no production code change)
- [ ] Style: styles updates (no production code change)
- [ ] Test: all about tests: adding, refactoring tests (no production code change)
- [ ] Other... Please describe:

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### Does this PR change client-test-data?

- [ ] Yes (don't forget to update the JS-LIB team about the change)
- [x] No

### CODE REVIEW

File: docs/02-dsp-ontologies/knora-base.md [hunk 1]
```diff
@@ -981,21 +981,17 @@ only if the value's class has some cardinality for that property.
 
 Knora supports, and attempts to enforce, the following cardinality constraints:
 
-`owl:cardinality 1`
+* `owl:cardinality 1`
+  : _Exactly One `1`_ - A resource of this class must have exactly one instance of the specified property.
```
[the remark this answers is not in the document]
Comment (seakayone) [reply docs/02-dsp-ontologies/knora-base.md:985]: TBH, I did not change the terminology and 'instance of' was use before and is also used throughout the rest of the whole paragraph. I am currently not inclined to change that.

[the remark this answers is not in the document]
Comment (irinaschubert) [reply docs/02-dsp-ontologies/knora-base.md:985]: If it is like this in the whole section, than it's better to leave it as it is.

File: webapi/src/main/scala/org/knora/webapi/responders/v2/OntologyResponderV2.scala [hunk 2]
```diff
@@ -1473,12 +1472,14 @@ final case class OntologyResponderV2(
   }
 
   /**
-   * Replaces a class's cardinalities with new ones.
+   * Replace cardinalities of a particular class.
```
Comment (mpro7) [inline webapi/src/main/scala/org/knora/webapi/responders/v2/OntologyResponderV2.scala:1475]:
```suggestion
   * Replaces cardinalities of a particular class.
```

**Review by mpro7 — APPROVED** [review]
`docs` should be a PR message prefix and task number missing at the end. Apart of that one small remark, check below.

**Pull request merged**